.editorconfig

```diff
@@ -10,4 +10,5 @@ indent_size = 2
 
 [*.md]
 indent_size = 4
-trim_trailing_whitespace = false
\ No newline at end of file
+trim_trailing_whitespace = false
+EOF
```

Comment:
EOF= End Of File.

La idea es que al final haya un enter, no una línea vacía sino un salto de línea.

Reply:
Exacto, quizas fue un poco confuso mi comentario, la idea es que la linea 14 este sin ningun caracter, ni EOF, ni tabs, ni espacios. A eso me refiero con que la ultima linea del archivo debe estar completamente vacia. Y comunmente para no escribir un comentario tan largo, comentamos en los reviews EOF.